src/mkCommand.nix

```diff
@@ -1,5 +1,10 @@
 {nixpkgs}: let
-  writeShellScript = currentSystem: nixpkgs.legacyPackages.${currentSystem}.writeShellScript;
-  mkCommand = currentSystem: args: args // {command = writeShellScript currentSystem "${args.name}" args.command;};
+  mkCommand = system: args: let
```

Comment:
This is now notably `currentSystem` and should keep that name. :smile:

Reply:
ah yeah, mistake during rebase